.github/workflows/build-and-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - ${{ matrix.build.runs-on }}
 
     container:
-      image: ghcr.io/${{ github.repository }}/tt-mlir-ci-ubuntu-22-04:latest
+      image: ghcr.io/${{ github.repository }}/tt-mlir-ci-ubuntu-22-04:cd3cfa1b9604877b1aa9a98d14346165b5d32733
```

Comment:
Do we want hard coded hashes?  Perhaps we can put in a variable if this is the case that can be shared between these jobs?

Reply:
No, this is just for testing. CI is using "latest" tag, just to make sure it's working before I promote the new image.
For future, I need to figure out how to automate this, build new image and run tests with it to make sure it is good before tagging.